### ISSUE TITLE
Can't Connect Philips Hue (IFTTT Workaround)

### ISSUE THREAD

**Issue by benblb** [amzn/alexa-avs-raspberry-pi#165]
I can't seem to connect my Echo to my Philips Hue lighting. Amazon Echo support said that Raspberry Echo does not currently connect to local devices (only cloud based) and suggested I try using IFTTT (https://ifttt.com/). It worked brilliantly! (Thanks, Erik!)

If anyone has a more permanent solution I'd love to hear it, but for now this is great.

**Comment by erikhopf-amzn** [comment]
Can you give me some more detail about your setup? Also, when you say Raspberry Echo, are you referring to this Pi project which uses the Alexa Voice Service?

Thanks!

**Comment by benblb** [comment]
Sure!

I followed the steps here (which I think are yours, thanks for those): https://github.com/amzn/alexa-avs-raspberry-pi.

I didn't add anything else and just have the Pi on my wifi network connected to a TV for now. I downloaded the Alexa app on my phone (Android) and it recognized my Pi as an Echo but can't directly connect my Hue lighting. Amazon support told me that local smart home devices are not supported with the homemade Echos which only support cloud-based smart home devices. I'm happy to answer any other questions that you have and hope this helped.
